src/transformers/models/fan/configuration_fan.py

```diff
@@ -0,0 +1,195 @@
+    "ksmcg/fan_base_18_p16_224": "https://huggingface.co/ksmcg/fan_base_18_p16_224/resolve/main/config.json",
+    "ksmcg/fan_large_24_p16_224": "https://huggingface.co/ksmcg/fan_large_24_p16_224/resolve/main/config.json",
+}
+
+
+# ISSUE: Move configuration to nvidia/fan
+class FANConfig(PretrainedConfig):
+    r"""
+    This is the configuration class to store the configuration of a [`~FANModel`]. It is used to instantiate an FAN
+    model according to the specified arguments, defining the model architecture. Instantiating a configuration with the
+    defaults will yield a similar configuration to that of the FAN
+    [ksmcg/fan_base_18_p16_224](https://huggingface.co/ksmcg/fan_base_18_p16_224) architecture.
+
+    Configuration objects inherit from [`PretrainedConfig`] and can be used to control the model outputs. Read the
+    documentation from [`PretrainedConfig`] for more information.
+
+
+    Args:
+        patch_size (int, defaults to 16):
+            Size of each patch to generated the embedding tokens from the image
+        hidden_size (`int`, *optional*, defaults to 448):
+            Dimension of the encoder layers and the pooler layer.
+        num_hidden_layers (`int`, *optional*, defaults to 18):
+            Number of hidden layers in the Transformer encoder.
+        num_attention_heads (`int`, *optional*, defaults to 8):
+            Number of attention heads for each attention layer in the Transformer encoder.
+        depths (`List[int]`, *optional*, defaults to None):
+            The number of layers in each encoder block, only applicable when using hybrid backbone (ConvNeXt).
+        eta (`float` defatults to 1.0):
+            Weight Initialization value for channel importance.
+        tokens_norm (`bool`, defaults to True):
+            Whether or not to apply normalization in the Class Attention block.
+        se_mlp (`bool`, defaults to False):
+            Wheter or not to use Squeeze-Excite in the FANEncoder layers MLP.
+        initializer_range (`float`, *optional*, defaults to 0.02):
+            The standard deviation of the truncated_normal_initializer for initializing all weight matrices.
+        img_size (`List[int]`, defaults to (224,224)):
+            The size of the images being passed to the model.
+        num_channels (`int`, defaults to 3):
+            Number of channels the input image has.
+        backbone (`string`, *optional*, defaults to None):
+            Wheter or not to use 'hybrid' backbone.
+        use_pos_embed ( `bool`, defaults to True):
+            Wheter or not to use positional_encoding in the embeddings.
+        mlp_ratio (`float`, defaults to 4.0):
+            Ratio of the size of the hidden layer compared to the size of the input layer of the Mix FFNs in the
+            encoder blocks.
+        qkv_bias (`bool`, defaults to True):
+            Whether or not to use bias in Query, Key and Value in attention layers.
```

Comment:
Please use the same format as other docstrings:
- \`type\`, \*optional\*, defaults to value (you're missing the optional here)
- True should be bteween `